### PR TITLE
Added option to configure frr_defaults traditional/datacenter 

### PR DESCRIPTION
**Pull request by elcomtik**
Added option to configure frr_defaults traditional/datacenter and updated README

### CODE REVIEW

**Comment by elcomtik** [comment]
The failed test on Ubuntu 16.04 is caused by dependency issues. This is probably temporary problem of apt repository.

File: defaults/main.yml [hunk 1]
```diff
@@ -12,12 +12,27 @@ frr_daemons:
   ripngd: false
   zebra: true
 
+# Define if we want to use traditional or datacenter FRR defaults
+# If undefined than current FRR default value is used implicitly (traditional)
+#frr_defaults: "datacenter"
```
Comment (nertwork) [inline defaults/main.yml:17]: Could we put something more in the readme for this? Not sure how we would go about doing that but might make it easier to follow there too ... like and example of using datacenter frr defaults?

Comment (elcomtik) [reply defaults/main.yml:17]: I can put there a link to official docs http://docs.frrouting.org/en/latest/basic.html#profiles. Actual values of timers can be found only in source code or by reviewing changes in running-config after applying it.

Comment (nertwork) [reply defaults/main.yml:17]: I think that would be best - other than that this looks great! Thanks for all your contributions!

**Comment by elcomtik** [comment]
@nertwork could you please finish the review

**Review by nertwork — APPROVED** [review]
Looks good - thank you!

**Pull request merged**